packages/scheduled-tasks/src/lib/types/ScheduledTaskStrategy.ts

```diff
@@ -0,0 +1,3 @@
+import type { ScheduledTaskRedisStrategy } from '../strategies';
+
+export type ScheduledTaskStrategy = ScheduledTaskRedisStrategy;
```

Comment:
This alias re-export is really useless tho innit. Just import `ScheduledTaskRedisStrategy` wherever you're currently importing `ScheduledTaskStrategy`

Reply:
Its preparation to extend upon when more strategies will be added. I still plan to add aws SQS to this PR